client/js/options.js

```diff
@@ -172,6 +173,8 @@ module.exports.initialize = () => {
 			if ($(this).prop("checked") && Notification.permission !== "granted") {
 				Notification.requestPermission(updateDesktopNotificationStatus);
 			}
+		} else if (name === "advanced") {
+			$(".container").find("[data-advanced]").toggle(options[name]);
```

Comment:
Isn't that something we'd want to show/hide with CSS instead?

Reply:
Well, we have to used javascript somewhere, and we either set a class on the window, and have the css work with that, or we do this. I'm not massively bothered as to which, but this involves less code/css total.

Reply:
Change this line to `$("#settings [data-advanced]").toggle(options[name]);`

Reply:
Done